### ISSUE TITLE
[macOS Ventura] Can't move player to another desktop space (Music Mode)

### ISSUE THREAD

**Issue by HiroiSekai** [iina/iina#4016]
**System and IINA version:**

- macOS 
- IINA 

**Expected behavior:**
When entering macOS Mission Control, the IINA window should stay visible to be able to drag it into a different space or desktop.

**Actual behavior:**
In Music Mode, the window disappears when entering Mission Control. You have to exit music mode, enter Mission Control, move the window, and then go back to Music Mode.

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**
Open IINA, enter Music Mode, and then try to enter Mission Control on the Mac. When you try to move the IINA player into another desktop or space, you can't because the window disappears.

- [ ] MPV does not have this problem.

**How often does this happen?**
Every time on the current version

**Comment by low-batt** [comment]
Did not reproduce for me.

On a 16" MacBook Pro with the M1 chip running the official release of macOS 13.0 Ventura I started playing a .wav file using IINA 1.3.0. I opened Mission Control. I grabbed the IINA window and dragged it to another space. It moved to the other space without a problem. I was then able to use Mission Control to move it back to the original space.

Some additional factor must be contributing to the problem.

When the IINA window disappears, did IINA crash? Was a crash report generated? Was the IINA application still running?

**Comment by HiroiSekai** [comment]
> Did not reproduce for me.
> 
> On a 16" MacBook Pro with the M1 chip running the official release of macOS 13.0 Ventura I started playing a .wav file using IINA 1.3.0. I opened Mission Control. I grabbed the IINA window and dragged it to another space. It moved to the other space without a problem. I was then able to use Mission Control to move it back to the original space.
> 
> Some additional factor must be contributing to the problem.
> 
> When the IINA window disappears, did IINA crash? Was a crash report generated? Was the IINA application still running?

Thanks for testing, this is a pivotal point but was your testing player on Music Mode when you tested it? Even with specific audio file playback, if the setting is off in IINA, it will just play in typical windowed form. That is no issue for me either, it is moveable in that mode. However, specifically in Music Mode, the window fades away when entering Mission Control. This isn't a crash or anything like that, all functions still work, just the window fades and you can't interact with it at all while in Music Mode.

If your testing was in Music Mode and can't be replicated, I'll record a screencap later tonight to upload here to visualize it better.

**Comment by low-batt** [comment]
Music Mode, like this, yes?:
![issue-4016](https://user-images.githubusercontent.com/86170219/198852128-f6d7d533-a113-412e-bfc3-793dbcaa7cdb.jpg)

What model Mac are you experiencing this on?

This is the 3rd strange problem that has been reported against the public release of Ventura. The other two problems are dimming when playing a HDR video in full screen and one display sometimes going black when IINA is playing on the other display.

There are significant changes related to graphics in Ventura. From [macOS Ventura is now available](https://www.apple.com/newsroom/2022/10/macos-ventura-is-now-available/):
> macOS Ventura brings new technologies like Metal 3 

Looking like these changes have introduced some odd problems.

**Comment by HiroiSekai** [comment]
Yup that's right! I just wanted to make sure. I'm using the base model 16" 2021 M1 Pro MacBook Pro. Now I'm a bit concerned if any other weird little tiny errors will pop up elsewhere, but nothing yet, just really this issue.
